gossip/src/restart_crds_values.rs

```diff
@@ -29,6 +29,25 @@ pub enum RestartLastVotedForkSlotsError {
     LastVotedForkEmpty,
 }
 
+#[derive(Serialize, Deserialize, Clone, PartialEq, Eq, AbiExample, Debug)]
+pub struct RestartHeaviestFork {
+    pub from: Pubkey,
+    pub wallclock: u64,
+    pub last_slot: Slot,
+    pub last_slot_hash: Hash,
+    pub observed_stake: u64,
+    pub total_epoch_stake: u64,
```

Comment:
why do we need this?
Isn't the `epoch_stakes` already known by all the nodes in the cluster?
https://github.com/solana-labs/solana/blob/1f00f8e29/runtime/src/bank.rs#L7744-L7746

Reply:
I thought it might be nice to distribute total_epoch_stake just in case some calculation is off. But you are right, if this stake is wrong we have bigger problems. Removed.